otherfile/font_wizard.py

```diff
@@ -58,7 +83,9 @@ def is_ascii(s):
         font_name = input("Please enter font name : ")
         if font_name in Font_List:
             print(Error6)
-        else:
+        elif font_name.lower() in INVALID_FONT_NAME:
```

Comment:
Minor suggestion from my part : 
I think here we can print `Error6` instead of a new error message because it's not possible for us to print all reserved fonts for users.
My suggestion:
```python
if font_name in Font_List or font_name.lower() in INVALID_FONT_NAME:
...
```